main.html

```diff
@@ -0,0 +1,127 @@
+                        <h1 class="hero__title">planet</h1>
+                        <p class="hero__content">Our planet is precious and vital for our beautiful lives,if we don't take care
+                            of it it will dissapear along with us.</p>
+                        <p class="hero__undernote">*Please take note that your actions matter more than you think.</p>
+                        <div class="hero__buttons">
+                            <a class="hero__button" href="#">Find out more</a>
+                            <a class="hero__button--transparent" href="#">Play video</a>
+                        </div>
+        </div>
+    </section>
+    <div class="about__container">
+        <div class="about__mission">
+            <h1 class="about__title">Our Mission</h1>
+            <div class="about__content">Our wildlife is important to us
+                if we don’t take care of it
+                maybe one day will turn
+                against us and we will die.
+            </div>
+            <div class="about__content">Our wildlife is important to us
+                if we don’t take care of it
+                maybe one day will turn
+                against us and we will die.
+            </div>
+            <div class="about__content">Our wildlife is important to us
+                if we don’t take care of it
+                maybe one day will turn
+                against us and we will die.
+            </div>
+        </div>
+        <div class="about__gallery">
+            <div class="about__photos">
+                <table class="tg">
+                    
+                      <tr>
+                        <td class="about__card td" rowspan="2">
+                            <div class="about__card">
+                            <div class="card__content">
+                                <h1 class="card__title">Wildlife</h1>              
+                                <p class="card__info">
+                                    Our wildlife is important to us
+                                        if we don’t take care of it
+                                        maybe one day will turn
+                                        against us and we will die.
+                                </p>
+                                <a class="hero__button--card" href="#">Find out more</a>
+                            </div>      
+                         </div>
+                        </td>
+                        <td class="photo__sheep td" colspan="3"><img class="photo__sheep" alt="herd of sheeps" src="about-sheep.jpg"></td>
+                    </tr>
+                    <tr>
+                        <td class="photo__lake td" colspan="2"><img class="photo__lake" alt="upper view of lake" src="about-lake.jpg"></td>
+                        <td class="photo__cliff td"><img class="photo__cliff" alt="upper view o a cliff with a castle on it" src="about-cliff.jpg"></td>

```

Comment:
nu ar trebui sa ai aceasi clasa si pe td si pe img

Reply:
Nu e aceeasi clasa,cea din tabel are "td" in plus,dar o sa modific.